Rakefile

```
@@ -5,4 +5,9 @@ require "rspec/core/rake_task"
 RuboCop::RakeTask.new
 RSpec::Core::RakeTask.new
 
-task default: %i[rubocop spec]
+desc "Lint Ruby"
+task :lint do
+  sh "bundle exec rubocop --format clang"
+end
+
+task default: %i[rubocop spec lint]
```

```suggestion
task default: %i[rubocop spec rubocop]
```

Looks like we've already got the rubocop task available so can simplify this a bit.

---

Oops just notice we already do rubocop at the start 😆  - I think we can just close this PR looks like it's already done.

---

How did I not notice that 🤯